### PR TITLE
TEN-170 amended single account to no longer redirect to PTA

### PR DESCRIPTION
**Pull request by ChrisHillCap**
awaiting journey test amendment

### CODE REVIEW

**Comment by ChrisHillCap** [comment]
https://github.com/hmrc/app-config-base/pull/7145

**Comment by ChrisHillCap** [comment]
https://github.com/hmrc/app-config-production/pull/14640

**Comment by ChrisHillCap** [comment]
https://github.com/hmrc/app-config-staging/pull/10504

**Comment by ChrisHillCap** [comment]
https://github.com/hmrc/tax-enrolment-assignment-journey-tests/pull/37

**Comment by ChrisHillCap** [comment]
https://github.com/hmrc/tax-enrolment-assignment-performance-tests/pull/13

File: README.md [hunk 1]
```diff
@@ -30,13 +30,18 @@ Run Journey Tests: see [here](https://github.com/hmrc/tax-enrolment-assignment-j
...
 
-| Path - internal routes prefixed by `/protect-tax-info` | Supported Methods | Type | Description |
-|-------|:-------------------:|:-------|-------------|
-|`/account-check`| GET | Internal | Endpoint get account check. |
-|`/multiple-accounts-check`| GET | Internal | Endpoint to get multiple accounts check. |
-|`/protect-tax-info/test-only/successful `| GET | Test | Endpoint to get a successful redirect. |
+| Path - internal routes prefixed by `/protect-tax-info` | Supported Methods | Type | Description                                   |
+|--------------------------------------------------------|:-------------------:|:-------|-----------------------------------------------|
+| `/account-check`                                       | GET | Internal | Endpoint get account check.                   |
+| `/multiple-accounts-check`                             | GET | Internal | Endpoint to get multiple accounts check.      |
```
Comment (Chrissystk) [inline README.md:39]: This one needs to be removed it doesn't exist :-)

File: README.md [hunk 2]
```diff
@@ -30,13 +30,18 @@ Run Journey Tests: see [here](https://github.com/hmrc/tax-enrolment-assignment-j
...
 ## API
 
-| Path - internal routes prefixed by `/protect-tax-info` | Supported Methods | Type | Description |
-|-------|:-------------------:|:-------|-------------|
-|`/account-check`| GET | Internal | Endpoint get account check. |
-|`/multiple-accounts-check`| GET | Internal | Endpoint to get multiple accounts check. |
-|`/protect-tax-info/test-only/successful `| GET | Test | Endpoint to get a successful redirect. |
+| Path - internal routes prefixed by `/protect-tax-info` | Supported Methods | Type | Description                                   |
+|--------------------------------------------------------|:-------------------:|:-------|-----------------------------------------------|
+| `/account-check`                                       | GET | Internal | Endpoint get account check.                   |
```
Comment (Chrissystk) [inline README.md:38]: Might be better to have /protect-tax-info in front of this

File: conf/app.routes [hunk 3]
```diff
@@ -10,8 +10,6 @@ GET        /keep-alive              uk.gov.hmrc.taxenrolmentassignmentfrontend.c
 
 GET        /                                                    uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.AccountCheckController.accountCheck(redirectUrl: String)
 
-#GET        /enrol-pt/enrolment-success-sa-user-id               uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.???
```
Comment (Chrissystk) [inline conf/app.routes:13]: This is required we need the story to implement it.

**Review by Chrissystk — CHANGES_REQUESTED** [review]

**Comment by Chrissystk** [comment]
I've ran all the tests and approved the other PRs so once the couple of comments have been address I will merge this in.

File: conf/app.routes (hunk 3 above)
Comment (ChrisHillCap) [reply conf/app.routes:13]: i dont think we should have commented code in our repos, it should be removed until implemented, we may go live without this journey

**Comment by ChrisHillCap** [comment]
amended

File: conf/app.routes (hunk 3 above)
Comment (Chrissystk) [reply conf/app.routes:13]: Should we be going live without this journey? The fact it is commented alerts us to the fact it hasn't been done. Had I not seen it I wouldn't have been alerted to the fact that a story hasn't been created for it.

Comment (ChrisHillCap) [reply conf/app.routes:13]: it should have been raised as a story in the first place, it should have never been a comment

Comment (Chrissystk) [reply conf/app.routes:13]: I never said that it should not have been raised as a story in the first place but for whatever reason a story hadn't been created and only by that comment being there was it picked up the fact it is has been missed. We clearly have differing views on this and are not going to agree so will merge your PR.

**Review by Chrissystk — APPROVED** [review]

**Pull request merged**